api/formatters/covjson.py

```diff
@@ -101,4 +101,7 @@ def _collect_data(ts_mdata, obs_mdata):
     )  # HACK: str -> float
     (times, values) = zip(*tuples)
 
+    if not ts_mdata.platform_name:
+        ts_mdata.platform_name = f"platform-{ts_mdata.platform}"
```

Comment:
I can't find where this is actually used? Does it end up in an output? None of the data json's are changed...

If it should be in the output, it should also be added to `response_fields_needed_for_data_api`.

Reply:
It was made based on @Teddy-1000 https://github.com/EURODEO/e-soh/pull/75#issuecomment-2012075933 and I thought it would be related to https://github.com/EURODEO/e-soh/issues/48.

It makes sure that the `ts_mdata` has a platform_name when collecting data needed to convert to `CoverageJSON`. But at the moment the converter does not use that part of `ts_mdata`  for a response, so it does not end up in any output. That is why I left it out from `response_fields_needed_for_data_api` in purpose for now to keep the message size at bay.

Reply:
I would prefer to leave it out then. To much risk of not updating `response_fields_needed_for_data_api` when this gets added to the converter, and always ending up with the default...

Reply:
Removed in https://github.com/EURODEO/e-soh/pull/75/commits/aae3ab5fff2c4c26087a48d4175eed431ab17e20